test/conference.js

```diff
@@ -183,6 +183,7 @@ contract('Conference', function(accounts) {
 
       await conference.isAttended(non_owner).should.eventually.eq(true)
       await conference.attended().should.eventually.eq(1)
+      await conference.totalAttended().should.eventually.eq(1)
```

Comment:
May not be the scope of this PR but can we either have `;` everywhere or nowhere?